### DISCUSSION TITLE
Different user interfaces on android chrome

### DISCUSSION THREAD

**Discussion by billlyd123** [webauthn4j/webauthn4j#777] [Q&A]
Hi I am noticing different UIs on android chrome between different websites. One of them is able to detect my Yubikey 5Ci and the other is not. Is there different ways of calling the API that generate different UIs?
<img width="416" alt="image" src="https://user-images.githubusercontent.com/30014142/231891676-461ee30e-e35b-422d-b982-28b6e30fc409.png">
![image](https://user-images.githubusercontent.com/30014142/231892072-b7643ae6-f64b-469c-94c7-4e8cc095f84e.jpg)

**Comment by ynojima** [comment]
Discussion is continued in https://github.com/webauthn4j/webauthn4j/issues/778. Closing
